src/app/app.component.ts

```diff
@@ -57,6 +57,9 @@ export class AppComponent implements OnInit {
     this.log.w('warn!');
     this.log.i('info');
     this.log.d('debug');
+    setTimeout(() => {
+      this.isWalletInitialized = this._modalsService.initializedWallet;
```

Comment:
This fix is invalid.. you'll have to use an observable.. The best is probably to use the state service to create some sort of state around whether or not the wallet is initialized